caesars-palace/blocks/header/header.js

```diff
@@ -1,8 +1,7 @@
 import { getMetadata, decorateIcons } from '../../scripts/lib-franklin.js';
 
-// media query match that indicates mobile/tablet width
-const isDesktop = window.matchMedia('(min-width: 1170px)');
-// const MAX_NAV_ITEMS_DESKTOP = 6;
+const smallDesktop = { isDesktop: window.matchMedia('(min-width: 1170px) and (max-width: 1439px'), maxItems: 7 };
+const largeDesktop = { isLargeDesktop: window.matchMedia('(min-width: 1440px)'), maxItems: 9 };
```

Comment:
Could these be converted to a config object?

eg
```
const screenConfig = Object.freeze({
  smallDesktop : {
    media: window.matchMedia('(min-width: 1170px) and (max-width: 1439px'),
    maxItems: 7,
},
  largeDesktop : {
},
})
```

Then it could be referenced as:

```
screenConfig.smallDesktop.media.matches
```